src/client/components/molecules/homeSections/second/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useContext, useEffect, useState } from 'react';
+import ItemList from '../../../atoms/itemList';
+import './style.scss';

```

Comment:
format and sort on every module you work.

`windows+shift+f` and `windows+shift+o` OR `alt+shift+f` and `alt+shift+o`(idk how is on windows)